addons/web_editor/static/src/js/wysiwyg/widgets/link_dialog.js

```diff
@@ -25,6 +25,8 @@ var LinkDialog = Dialog.extend({
     }),
 
     /**
+     * Uses a boolean 'linkInfo.isButton' to adapt the changes to button elements.
```

Comment:
Strange comment. At best you can add `@param` line at the end of the doc comment to describe the `isButton` parameter explicitely but this should not be the whole doc of the constructor.